conformance/05_referrers.go

```diff
@@ -0,0 +1,306 @@
+package conformance
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/bloodorangeio/reggie"
+	g "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var test05Referrers = func() {
+	g.Context(titleReferrers, func() {
+
+		g.Context("Setup", func() {
+
```

Comment:
We need to push the empty JSON descriptor blob too.

Reply:
Will do but not clear if registries expect this blob to be present. They are likely going to just compare digest/media-type and ignore since a very special case.

Reply:
Fixed.